File: CyFile-Android/app/src/main/java/at/tugraz/tc/cyfile/crypto/exceptions/KeyVaultServiceException.java
package at.tugraz.tc.cyfile.crypto.exceptions;

public class KeyVaultServiceException extends CryptoException {

    public KeyVaultServiceException() {
    }

    public KeyVaultServiceException(String message) {
        super(message);
    }

    public KeyVaultServiceException(String message, Throwable cause) {
        super(message, cause);
    }

    public KeyVaultServiceException(Throwable cause) {
        super(cause);
    }

}

File: CyFile-Android/app/src/main/kotlin/at/tugraz/tc/cyfile/crypto/exceptions/KeyVaultServiceException.kt
package at.tugraz.tc.cyfile.crypto.exceptions


open class KeyVaultServiceException : RuntimeException {

    constructor()

    constructor(message: String) : super(message)

    constructor(message: String, cause: Throwable) : super(message, cause)

    constructor(cause: Throwable) : super(cause)

}